network-api/networkapi/wagtailpages/pagemodels/customblocks/tito_widget.py

```diff
@@ -20,3 +23,22 @@ class TitoWidgetBlock(blocks.StructBlock):
     class Meta:
         icon = "form"
         template = "wagtailpages/blocks/tito_widget_block.html"
+
+    def get_context(self, request, parent_context=None):
+        context = super().get_context(request, parent_context=parent_context)
+        request_language_code = get_language_from_request(context["request"])
+        context["tito_widget_lang_code"] = self.get_widget_language_code(request_language_code)
+
+        return context
+
+    # Checking if the user's requested language is currently supported by Tito.
+    # If not, default to English, to prevent the Tito widget from crashing due to an unsupported language.
+    # For more info see: https://github.com/mozilla/foundation.mozilla.org/issues/9790
+    def get_widget_language_code(self, request_language_code):
```

Comment:
What do you think about turning the comment into a docstring for the function?

Reply:
Sounds good to me!